lib/content-services/src/lib/search/components/reset-search.directive.spec.ts

```diff
@@ -23,7 +23,7 @@ import { SearchFacetFiltersService } from '../services/search-facet-filters.serv
 import { SearchQueryBuilderService } from '../services/search-query-builder.service';
 
 @Component({
-    template: `<button adf-reset-search></button>`
+    template: `<button adf-reset-search aria-label="reset-search"></button>`
```

Comment:
There is no need to fix accessibility for unit test mocks because accessibility is needed only for users and unit test mocks won't be seen by users. I think you missed that point in jira's task, there is point which mentions that and shows which solution should be applied to skip fixing accessibility in unit testing: 
<img width="944" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/f4cff6ae-e99f-40ff-b32b-4d65bcbdfaf5">